src/braket/pennylane_plugin/ops.py

```diff
@@ -415,61 +319,6 @@ def _matrix(cls, *params):
         )
 
 
-class XX(Operation):

```

Comment:
[Available in PennyLane](https://github.com/PennyLaneAI/pennylane/blob/ca227436391adf10df1f946cc727a9abb36673eb/pennylane/ops/qubit.py#L1774)